tests/unit/Makefile.am

```diff
@@ -4,8 +4,9 @@
 # See LICENSE.txt for license information
 #
 
-AM_CPPFLAGS = -I$(top_srcdir)/include
-LDADD = $(top_builddir)/src/libinternal_net_plugin.la
+AM_CPPFLAGS = -I$(top_srcdir)/include $(CUDA_CPPFLAGS)
```

Comment:
same comments as in src/Makefile.am

Reply:
See response -- I first started by adding them to the targets that actually needed them. The combination of libtool + mpicc + ??? requires it in all of these places. I fought this for over an hour before admitting defeat and putting it in the top targets.